src/containers/NamespacesDropdown/NamespacesDropdown.js

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { Dropdown } from 'carbon-components-react';
+
+import { fetchNamespaces } from '../../actions/namespaces';
+import { getNamespaces } from '../../reducers';
+
+class NamespacesDropdown extends React.Component {
+  componentDidMount() {
+    this.props.fetchNamespaces();
```

Comment:
namespaces should always be available in the dashboard so no need to request them here

Reply:
How frequently are the namespaces updated? Can I assume that the namespaces will be up-to-date when the component mounts?

Reply:
Namespaces are only retrieved on page load at the moment, but once we address https://github.com/tektoncd/dashboard/issues/91 they could be kept up to date as changes are made.

Do we expect to see namespaces created / deleted frequently?

Reply:
No, I don't think so. On page loads will definitely be good enough for this.